packages/x-components/src/x-modules/url/components/url-handler.vue

```diff
@@ -156,7 +163,10 @@
     protected emitEvents(): void {
       const { all, extra } = this.parseUrlParams();
       const metadata = this.createWireMetadata();
-      this.$x.emit('ParamsLoadedFromUrl', all, metadata);
+      this.preventSpecialKey(all.query);
+      if (!this.hasSpecialKeys) {
```

Comment:
Just curious, I think I like the approach, but why are we preventing to emit the event if we have special keys instead of removing them, and send send it without them?

Reply:
Do you prefer to remove the special keys instead of the whole query? I mean, if you have this query `round <hola> shirt`, it would be transformed into `round shirt` and sent into the `ParamsLoadedFromUrl`event.

I use this approach because, in the task description, I understood that we have to clean/remove the whole query. But for me makes more sense to remove only the special characters and the string inside them.